### PR TITLE
add fileImageController

### PR DESCRIPTION
**Pull request by MariyaSin**

### CODE REVIEW

File: src/ImageService.Business/Commands/File/GetFileImageCommand.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        extension = dbImageProject.Extension;
+      }
+      else if (directory == ImageDirectory.User)
+      {
+        DbImageUser dbImageUser = _userRepository.Get(imageId);
+        content = dbImageUser.Content;
+        extension = dbImageUser.Extension;
+      }
+
+      new FileExtensionContentTypeProvider().TryGetContentType(extension, out var contentType);
```
Comment (BelehovEgor) [inline src/ImageService.Business/Commands/File/GetFileImageCommand.cs:59]: it return bool, why do you ignore it, what happened if you could not get content type

File: src/ImageService.Business/Commands/File/Interfaces/IGetFileImageCommand.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using LT.DigitalOffice.ImageService.Models.Dto.Enums;
+using LT.DigitalOffice.Kernel.Attributes;
+
+namespace LT.DigitalOffice.ImageService.Business.Commands.Interfaces
+{
+  [AutoInject]
+  public interface IGetFileImageCommand
+  {
+    (byte[], string) Execute(Guid imageId, ImageDirectory directory);
```
Comment (BelehovEgor) [inline src/ImageService.Business/Commands/File/Interfaces/IGetFileImageCommand.cs:10]: maybe better to add name of tuple elements

File: src/ImageService.Models.Dto/Enums/ImageDirectory.cs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+﻿namespace LT.DigitalOffice.ImageService.Models.Dto.Enums
+{
+  public enum ImageDirectory
```
Comment (BelehovEgor) [inline src/ImageService.Models.Dto/Enums/ImageDirectory.cs:3]: Strange name of enum

Comment (MariyaSin) [reply src/ImageService.Models.Dto/Enums/ImageDirectory.cs]: renamed

File: src/ImageService.Business/Commands/File/GetFileImageCommand.cs [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+      _projectRepository = projectRepository;
+      _userRepository = userRepository;
+    }
+
+    public (byte[] content, string extension) Execute(Guid imageId, ImageType directory)
+    {
+      string content = null;
+      string extension = null;
+
+      if (directory == ImageType.Message)
```
Comment (VLiamin) [inline src/ImageService.Business/Commands/File/GetFileImageCommand.cs:34]: switch case?

**Review by VLiamin — APPROVED** [review]

File: src/ImageService.Business/Commands/File/GetFileImageCommand.cs (hunk 4 above)
Comment (MariyaSin) [reply src/ImageService.Business/Commands/File/GetFileImageCommand.cs:34]: here you can both

**Review by BelehovEgor — APPROVED** [review]

**Review by ndoriomedov — APPROVED** [review]

**Review by Glunky — APPROVED** [review]

**Pull request merged**